coala_quickstart/Constants.py

```diff
@@ -10,6 +10,27 @@
     "Java": {"JavaPMD", "CheckstyleBear"},
     "Python": {"PycodestyleBear"}}
 
+ALL_CAPABILITIES = {
+    'Code Simplification',
+    'Commented Code',
+    'Complexity',
+    'Documentation',
+    'Duplication',
+    'Formatting',
+    'Grammar',
+    'Memory Leak',
+    'Missing Import',
+    'Redundancy',
+    'Security',
+    'Smell',
+    'Spelling',
+    'Syntax',
+    'Undefined Element',
+    'Unreachable Code',
+    'Unused Code',
+    'Variable Misuse'
+}
+
```

Comment:
hmm, I've included this to avoid the overhead of generating this list dynamically during runtime. Also as this list is language independent, there'd be all the bears every time even if we generate it dynamically